src/Cameras/VR/vrExperienceHelper.ts

```diff
@@ -616,6 +616,13 @@ export class VRExperienceHelper {
         return this._vrDeviceOrientationCamera;
     }
 
+    /**
+     * The html button that is used to trigger entering into VR.
+     */
+    public get vrButton(): HTMLButtonElement | undefined {

```

Comment:
Can you use Nullable<HTMLButtonElement > (and flat the internal button the same way?